users/models.py

```diff
@@ -0,0 +1,64 @@
+from django.db import models
+
+
+class User(models.Model):
+    name          = models.CharField(max_length=30)
+    kakao_id      = models.BigIntegerField(default=0, null=True)
+    email         = models.CharField(max_length=200)
+    phone_number  = models.CharField(max_length=50, null=True)
+    profile_image = models.CharField(max_length=500)
+    gender        = models.CharField(max_length=30)
+    age           = models.PositiveIntegerField()
+    myclasses     = models.ManyToManyField('products.Product', through="users.MyClass", related_name='user_myclass')
+    reviews       = models.ManyToManyField('products.Product', through="users.Review", related_name='user_review')
+    likes         = models.ManyToManyField('products.Product', through="users.Like", related_name='user_like')
+    class Meta:
+        db_table = 'users'
+
+class Creator(models.Model):
+    nickname      = models.CharField(max_length=30)
+    kakao_id      = models.CharField(max_length=400)
+    email         = models.CharField(max_length=200)
+    phone_number  = models.CharField(max_length=50, null=True)
+    profile_image = models.CharField(max_length=500)
+    introduction  = models.CharField(max_length=1000)
+    carrer        = models.CharField(max_length=1000)
+
+    class Meta:
+        db_table = 'creators'
```

Comment:
처음부터 Creator로만 등록할 수 있나요?
처음에는 일반 유저 상태에서 Creator로 등록하는게 기획 아닌가요?

기존 유저가 Creator로 전환되는 기획이라면 아래와 같이 수정되야 될 것 같습니다.

```suggestion
class Creator(models.Model):
    user = models.ForeignKey('Users')
    email         = models.CharField(max_length=200)
    phone_number  = models.CharField(max_length=50, null=True)
    profile_image = models.CharField(max_length=500)
    introduction  = models.CharField(max_length=1000)
    carrer        = models.CharField(max_length=1000)

    class Meta:
        db_table = 'creators'
```

Reply:
nickname은 유저의 이름과 강의할때의 필명이 다른 경우가 있을 수 있어서 필요한 필드입니다. 조언 감사합니다!